src/backend/tests/change-requests.test.ts

```diff
@@ -0,0 +1,248 @@
+import request from 'supertest';
+import express from 'express';
+import prisma from '../src/prisma/prisma';
+import { batman, superman, wonderwoman } from './test-data/users.test-data';
+import changeRequestsRouter from '../src/routes/change-requests.routes';
+import {
+  changeBatmobile,
+  redesignWhip,
+  redesignWhipScopeCR,
+  redesignWhipWBSElement,
+  solutionToRedesignWhip,
+  unreviewedCr,
+  whipWorkPackage
+} from './test-data/change-requests.test-data';
+import { project1 } from './test-data/projects.test-data';
+
+const app = express();
+app.use(express.json());
+app.use('/', changeRequestsRouter);
+
+describe('Change-Requests', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('reviewer not found errors', async () => {
+    jest.spyOn(prisma.user, 'findUnique').mockResolvedValue(null);
+    const response = await request(app).post('/review').send(changeBatmobile);
```

Comment:
we shouldn't be sending change request objects for any of these. we should be sending a payload object and the fields for that you can find in teh routes folder